monero-outreach-docs/translations/fr/merchants/how_to_accept_monero_fr.md

```diff
@@ -0,0 +1,85 @@
+
+Si un délai de 4 à 12 minutes n'est pas compatible avec votre modèle d'affaire, vous pouvez faire une transaction à zéro confirmations en une seconde. Plutôt que d'attendre pour une première confirmation, vous pouvez consulter le pool de transactions pour vérifier que la transaction existe et y est inscrite. Il y a un risque associé à cette manière de procéder car la transaction n'est pas encore confirmée, mais une attaque de double-dépense est peu probable si la personne est physiquement devant vous. Pour des transactions commerciales de petite taille, l'avantage d'une transaction rapide vaut habituellement ce faible risque. Si quelque chose vous paraît anormal dans la transaction, vous pouvez toujours choisir d'attendre la première confirmation si les circonstances le justifient.
+
+### _Identification des Transactions_
+
+ Il devient rapidement évident, lorsque vous commencez à accepter le paiement en Monero, que vous ne pouvez pas automatiquement connaître l'adresse d'origine des transactions puisque celles-ci sont privées. Tout dépendant de votre domaine d'activité, l'association entre une transaction et un paiement reçu pourrait être critique ou superflue. Il existe plusieurs techniques simples pour vous aider à identifier positivement vos paiements XMR.
+
+ La première est la plus simple: prenez-en note. Si vous faites affaire à un bas volume de transactions, vous pouvez ajouter le numéro de facture ou de transaction comme une note dans votre portefeuille lorsque vous recevez les fonds. Vous pourriez aussi prendre note du numéro d'identifiant (ID) de transaction Monero généré par votre terminal de point de vente ou par votre système comptable.
+
+ Toutefois, si vous devez absolument lier le paiement d'un client à une transaction spécifique, l'utilisation des sous-adresses est la solution à privilégier. Une sous-adresse est une adresse à usage unique, et puisqu'elles sont gratuites et illimitées, vous pouvez générer une sous-adresse pour chaque client afin de suivre chaque paiement. Cette fonctionnalité est également utile pour éviter de divulguer votre adresse principale.
+
+###_Transactions Multisig_
+
+Les transactions multisig requièrent une signature de plusieurs parties avant d'être annoncée au réseau. Par exemple, vous pourriez préparer une transaction où les fonds ne seraient relâchés jusqu'à ce que l'acheteur et le vendeur donnent conjointement leur accord. Sans explorer les détails techniques des transactions multisig ici, vous pouvez trouver plus de renseignements au lien suivant: [getmonero.org/resources/user-guides/multisig-messaging-system.html](https://www.getmonero.org/resources/user-guides/multisig-messaging-system.html)
```

Comment:
Change the link to French getmonero link: https://www.getmonero.org/fr/resources/user-guides/multisig-messaging-system.html
[getmonero.org/resources/user-guides/multisig-messaging-system.html](https://www.getmonero.org/fr/resources/user-guides/multisig-messaging-system.html)

Reply:
Link changed